app/views/service_providers/_form.html.erb

```diff
@@ -350,7 +350,9 @@ B7xxt8BVc69rHV15A0qyx77CLSj3tCx2IUXVqRs5mlvA==
 
   <fieldset class="usa-fieldset-inputs custom-help-text">
     <legend>Custom help text</legend>
-    <p>You can specify help text in these different areas to alert users of important information for logging into your app</p>
+    <p>You can specify help text in these different areas to alert users of important information for logging into your app. Permitted HTML elements include the following:
+      <code>p</code>, <code>br</code>, <code>ol</code>, <code>ul</code>, <code>li</code>, <code>b</code>, <code>strong</code>, and <code>em</code>.
```

Comment:
I wonder if we should create a constant for the allowed tags, to share between here and `sanitize_section`.

```suggestion
      <%= safe_join(ServiceProvider::ALLOWED_HTML_ELEMENTS.map { |tag_name| content_tag(:code, tag_name) }, ', ') %>.
```

Reply:
great idea @aduth! I'll make that happen